### PR TITLE
Clean up run version bump script

### PR DESCRIPTION
**Pull request by danfinlay**
Per feedback on that approved & merged PR.

### CODE REVIEW

File: development/run-version-bump.js [hunk 1]
```diff
@@ -7,25 +7,28 @@ const changelogPath = path.join(__dirname, '..', 'CHANGELOG.md')
...
+    const manifestString = JSON.stringify(newData.manifest, null, 2)
 
-  return newData.version
-})
-.then((version) => console.log(`Bumped ${bumpType} to version ${version}`))
-.catch(console.error)
+    await writeFile(changelogPath, newData.changelog)
+    await writeFile(manifestPath, manifestString)
+
+    return `Bumped ${bumpType} to version ${newData.version}`
```
Comment (kumavis) [inline development/run-version-bump.js]: i would just console.log to make this clearer

File: development/run-version-bump.js [hunk 2]
```diff
@@ -7,25 +7,28 @@ const changelogPath = path.join(__dirname, '..', 'CHANGELOG.md')
...
+.then(console.log)
+.catch(console.error)
 
-readFile(changelogPath)
-.then(async (changeBuffer) => {
-  const changelog = changeBuffer.toString()
+async function start() {
 
-  const newData = await versionBump(bumpType, changelog, manifest)
+  readFile(changelogPath)
```
Comment (kumavis) [inline development/run-version-bump.js]: the point of moving this into an async fn is so you could use an `await` here : )

File: development/run-version-bump.js [hunk 3]
```diff
@@ -7,25 +7,28 @@ const changelogPath = path.join(__dirname, '..', 'CHANGELOG.md')
 const manifestPath = path.join(__dirname, '..', 'app', 'manifest.json')
 const manifest = require('../app/manifest.json')
 const versionBump = require('./version-bump')
-
 const bumpType = normalizeType(process.argv[2])
 
+start()
+.then(console.log)
```
Comment (kumavis) [inline development/run-version-bump.js]: this is verbatim what i said but i guess we dont need the `.then()`

File: development/run-version-bump.js (hunk 2 above)
Comment (danfinlay) [reply development/run-version-bump.js]: oh lol I copy pasted good call.

**Review by kumavis — APPROVED** [review]
so clean 👍

**Pull request merged**